Demo/Demo.xcodeproj/project.pbxproj

```diff
@@ -2064,7 +2064,7 @@
 				GCC_WARN_UNINITIALIZED_AUTOS = YES_AGGRESSIVE;
 				GCC_WARN_UNUSED_FUNCTION = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
-				IPHONEOS_DEPLOYMENT_TARGET = 15.0;
+				IPHONEOS_DEPLOYMENT_TARGET = 16.6;
```

Comment:
shouldn't it be 16.0?

Reply:
That's the value it sets when selecting `iOS 16` from the dropdown. This is only for the Demo, so I won't bother fixing it manually.

<img width="1162" alt="Screenshot 2024-10-16 at 14 38 48" src="https://github.com/user-attachments/assets/6968d485-e603-4fc6-a2c1-ed1d2756a25f">